src/components/link/link.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+import styles from './link.module.scss';
+
+interface Props {
+  className?: string;
+  href: string;
+  children: ReactNode;
+}
+
+const Link = (props: Props) => {
+  const { className, href, children } = props;
+
+  return (
+    <a href={href} className={classNames(className, styles.link)} target="_blank" rel="noopener noreferrer">
```

Comment:
I wonder if it's worth calling this `<ExternalLink />` instead since it's only intended for opening external links in new tabs?

Reply:
Yeah, I agree.